index.bs

```diff
@@ -260,7 +260,7 @@ spec:url; type:dfn; text:scheme
 
     1. Let |portalBrowsingContext| be the [=guest browsing context=] of [=this=].
 
-        If no such context exists, throw an "{{InvalidStateError}}" {{DOMException}}.
+        If it is null, throw an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
Right below this, you have a similar "if no such context exists" for document browsing context, which has these days also moved to a null-or-BC model. Might as well fix while here.

Reply:
Done.